listings/ch08-common-collections/listing-08-22/src/main.rs

```diff
@@ -7,7 +7,6 @@ fn main() {
 
     let mut map = HashMap::new();
     map.insert(field_name, field_value);
-    // field_name and field_value are invalid at this point, try using them and
-    // see what compiler error you get!
+    // field_name 和 field_value 在這之後就不能使用了，嘗試使用它們並看看編譯器回傳什麼錯誤
```

Comment:
nit: 我在想也許這樣更口語一點？ 🤔  cc @wusyong @weihanglo 

```suggestion
    // field_name 和 field_value 在這之後就不能使用了，你可以試著使用它們並看看編譯器回傳什麼錯誤
```

Reply:
both are great!

Reply:
看起來有比較通順，已修改！